src/physics/physics.h

```diff
@@ -51,6 +51,11 @@ class PhysicsBase
     virtual std::array<dealii::Tensor<1,dim,real>,nstate> convective_numerical_split_flux (
             const std::array<real,nstate> &soln_const, const std::array<real,nstate> &soln_loop) const = 0;
 
+/// Convective Numerical Split Flux for split form
+    virtual std::array<dealii::Tensor<1,dim,real>,nstate> convective_surface_numerical_split_flux (
+                const std::array< dealii::Tensor<1,dim,real>, nstate > &surface_flux,
+                const std::array< dealii::Tensor<1,dim,real>, nstate > &flux_interp_to_surface) const = 0;
```

Comment:
consider removing the `=0` and giving the default function definition in `physics.cpp` to reduce code repetition in the derived classes

Reply:
I left it because this will be fixed by surface splitting. I will open a ticket once github lets me. currently I am getting an error saying issue was submitted to quickly.

Reply:
fixed via issue #128